List_of_Users_Project/userList.py

```diff
@@ -0,0 +1,70 @@
+import requests
+from datetime import datetime, timedelta
+import time
+import json
+import calendar
+import csv
+import sys
+import os
+import argparse
+import urllib3
+
+#to disable warnings for not verifying host
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+class List:
+    def __init__(self, cli_args):
+        arg_parser = self.__initalize_argument_parser()
+        self.args = arg_parser.parse_args(cli_args)
+        self.api_url = "https://api2.rhombussystems.com"
+        self.api_sess = requests.session()
+        self.api_sess.headers = {
+            "Accept": "application/json",
+            "x-auth-scheme": "api-token",
+            "Content-Type": "application/json",
+            "x-auth-apikey": self.args.APIkey}
+
+    @staticmethod
+    def __initalize_argument_parser():
+        parser = argparse.ArgumentParser(
+            description= "Gets a report of all of the Users and their emails.")
+        #aruements avaiable for the user to customize
+        parser.add_argument('APIkey', type=str, help='Get this from your console')
+        parser.add_argument('-c', '--csv', type=str, help= 'Name the csv file', default='csvFile.csv')
+        parser.add_argument('-p', '--path', type=str, help='Path to where the csv will go', default=os.getcwd())
```

Comment:
You still don't need both -c and -p arguments, there should just be one argument to tell the program where to try to write the CSV